src/containers/PointOfCareApp.jsx

```diff
@@ -273,6 +313,114 @@ export class PointOfCareApp extends Component {
         }
     }
 
+    // Update shortcuts and update patients accordingly
+    handleShortcutUpdate = (s) => {
+        let p = this.state.patient;
+        let note = this.state.openClinicalNote;
+        s.updatePatient(p, this.contextManager, note);
+    }
+
+    newCurrentShortcut = (shortcutC, shortcutType, shortcutData, updatePatient = true, source) => {
+        let newShortcut = this.shortcutManager.createShortcut(shortcutC, shortcutType, this.state.patient, shortcutData, this.handleShortcutUpdate);
+        newShortcut.setSource(source);
+        const errors = newShortcut.validateInCurrentContext(this.contextManager);
+        if (errors.length > 0) {
+            errors.forEach((error) => {
+                console.error(error);
+            });
+            newShortcut = null;
+        } else {
+            newShortcut.initialize(this.contextManager, shortcutType, updatePatient, shortcutData);
+        }
+        return newShortcut;
+    }
+
+    closeNote = () => {
+        this.handleModalClose();
+    }
+
+    getNoteModalStyle = () => {
+        const clinicalNoteStyle = {
+            width: '50%',
+            height: '75%',
+        };
+        return Object.assign(getModalStyle(), clinicalNoteStyle);
```

Comment:
Can we get a comment on this explaining that we're merging the default modal styling with the clinicalNote specific styles.